pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -238,7 +244,15 @@ stdenv.mkDerivation (finalAttrs: {
       attrPath = "gtk3";
       freeze = true;
     };
-    tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
+    tests = {
+      inherit
+        clutter
+        gtk-sharp-3_0
+        gtkmm3
+        gtkspell3
```

Comment:
All the packages except for `gtkmm3` are dead.

Reply:
We'd still prefer they didn't break though, right? There are still quite a few packages depending on them..

If you have some better suggestions, please add them. I tend to aim for highly-depended-upon packages here, hence going for libraries rather than applications.

Reply:
I would prefer to just remove the `clutter` and `gtk-sharp-3_0` when they break (or even sooner when I feel particularly bored). Would not call those two highly-depended-upon either.